src/EscaperInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Escaper;
+
+/**
+ * Interface for context specific methods for use in secure output escaping
+ */
+interface EscaperInterface
+{
+    /**
+     * Return the encoding that all output/input is expected to be encoded in.
+     *
+     * @return non-empty-string
+     */
+    public function getEncoding();
```

Comment:
I don't think `getEncoding` should be included in the interface. It's configuration information that as a user, you already know, and, is rarely anything other than 'utf-8'